### ISSUE TITLE
Problems executing in linux

### ISSUE THREAD

**Issue by seraasch** [morgant/finishGoogleCodeGitHubWikiMigration#4]
Thanks for this!

Don't know if this is my company's oddball linux installation or not, but I had several issues executing this. I have it fixed now if you're interested.

Was this developed on a Mac?

**Comment by morgant** [comment]
@seraasch Yes, this was developed on OS X, so there might be some differences when running under Linux (esp. regexes in `sed`). I'd definitely be interested in what you had to change to make it work so I can try to improve Linux support.

**Comment by seraasch** [comment]
Happy to send it to you. I'm a GitHub noob, so how should I get it to you?

I've made changes to the repo I cloned (still need to commit them)...

**Comment by seraasch** [comment]
OK. I've forked, updated, and pushed my changes into my branch.

**Comment by anarcat** [comment]
see #5
